src/service/lootService.ts

```diff
@@ -239,6 +243,42 @@ const lootTemplates: loot.LootTemplate[] = [
         emote: "🏆",
         asset: "assets/loot/17-achievement.png",
     },
+    {
+        id: LootTypeId.GME_AKTIE,
+        weight: 5,
+        displayName: "GME-Aktie",
+        titleText: "Eine GME Aktie",
+        description: "Der squeeze kommt bald!",
+        emote: "📈",
```

Comment:
```suggestion
        emote: "📉",
```

Reply:
Hmm i had the idea for bitcoin and would much rather assign the emote to the bitcoin.

Reply:
`:peepoNotStonks:`?

Reply:
Ah they're not identifiers so fine by me.